src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -70,7 +72,7 @@
     :banner              (resources/get-image :discover)
     :accessibility-label :communities-home-discover-card}})
 
-(defn home
+(defn- f-view-internal
   []
```

Comment:
here this should be making use of `theme` prop which we have now thanks to `with-theme` HOC.
i.e 
```
(defn- f-view-internal
  [{:keys [theme]}]
```
and then this needs to get passed along to `theme-value` function.

`(theme/theme-value value-1 value-2 theme)`